autoload/ale/linter.vim

```diff
@@ -22,6 +22,7 @@ let s:default_ale_linter_aliases = {
 " Only cargo is enabled for Rust by default.
 let s:default_ale_linters = {
 \   'csh': ['shell'],
+\   'help': [],
```

Comment:
It _might_ be okay to remove this line and just enable linting for Vim help files by default. I'm not entirely sure about this either way.